test/srt/test_fused_moe.py

```diff
@@ -45,20 +44,49 @@ def get_tolerance(self, dtype):
         else:
             return 1e-2, 1e-2  # Default values for other types
 
-    def torch_naive_moe(self, a, w1, w2, score, topk):
+    def torch_naive_moe(
+        self,
+        a,
+        w1,
+        w2,
+        score,
+        topk,
+        w1_scale=None,
+        w2_scale=None,
+        a1_scale=None,
+        a2_scale=None,
+    ):
         B, D = a.shape
         a = a.view(B, -1, D).repeat(1, topk, 1).reshape(-1, D)
         out = torch.zeros(B * topk, w2.shape[1], dtype=a.dtype, device=a.device)
         score = torch.softmax(score, dim=-1, dtype=torch.float32)
         topk_weight, topk_ids = torch.topk(score, topk)
         topk_weight = topk_weight.view(-1)
         topk_ids = topk_ids.view(-1)
-        for i in range(w1.shape[0]):
+
+        if w1.dtype == torch.float8_e4m3fn:
+            w1_orig = w1.to(a.dtype)
```

Comment:
nit: wondering whether the name can be slightly optimized

Reply:
Make sense, I have renamed it to `w1/2_compute`.